contrib/exporters/secadvisor/mod/transform_test.go

```diff
@@ -130,9 +203,10 @@ func Test_Transform_basic_flow(t *testing.T) {
 	assertEqual(t, "80", secAdvFlow.Transport.B)
 }
 
-func Test_Transform_UpdateCount_increses(t *testing.T) {
+func Test_Transform_UpdateCount_increases(t *testing.T) {
```

Comment:
in go the standard notation is CamelCase so name should rather be TestTransformUpdateCountIncreases()